lib/ansible/executor/process/worker.py

```diff
@@ -77,7 +79,10 @@ def __init__(self, rslt_q, task_vars, host, task, play_context, loader, variable
                 fileno = sys.stdin.fileno()
                 if fileno is not None:
                     try:
-                        self._new_stdin = os.fdopen(os.dup(fileno))
+                        if PY3: # needed for recognizing Enter in 'pause' module
+                            self._new_stdin = os.fdopen(os.dup(fileno), newline='\r\n')
```

Comment:
Is the intention to *only* recognize Enter if it sends the string "\r\n"?  Or is the intention to recognize any of "\r", "\n", or "\r\n"?  Documentation says ( https://docs.python.org/3/library/functions.html#open ) that newline=None or newline="" should be used if the latter is meant.